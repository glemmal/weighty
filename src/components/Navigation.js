import React, { Component } from 'react';
import LogoImage from '../assets/weighty.svg';

class Navigation extends Component {
  render() {
    return (
      <nav className="navbar is-danger" role="navigation" aria-label="main navigation">
        <div className="navbar-brand">
          <a className="navbar-item" href="https://bulma.io">
            <img src={LogoImage} alt="Weighty logo" width="112" height="28" />
          </a>
        </div>
      </nav>
    );
  }
}

export default Navigation