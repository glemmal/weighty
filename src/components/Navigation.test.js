import React from 'react';
import Navigation from './Navigation';

import {shallow} from 'enzyme';

test('Navigation is rendered correclty', () => {
  const component = shallow(<Navigation />);
  expect(component.exists());
});