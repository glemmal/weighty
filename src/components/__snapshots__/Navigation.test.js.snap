// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders correctly 1`] = `
<nav
  aria-label="main navigation"
  className="navbar is-danger"
  role="navigation"
>
  <div
    className="navbar-brand"
  >
    <a
      className="navbar-item"
      href="https://bulma.io"
    >
      <img
        alt="Weighty logo"
        height="28"
        src="weighty.svg"
        width="112"
      />
    </a>
  </div>
</nav>
`;
